docs/api/channels.rst

```diff
@@ -106,3 +310,122 @@ Exemplary Response
             }
         }
     }
+
+Collection of Channels
+-----------------------
+
+To retrieve a paginated list of channels you will need to call the ``/api/v1/channels/`` endpoint with the ``GET`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/channels/
+
++---------------+----------------+-------------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                       |
++===============+================+===================================================================+
+| Authorization | header         | Token received during authentication                              |
++---------------+----------------+-------------------------------------------------------------------+
+| page          | query          | *(optional)* Number of the page, by default = 1                   |
++---------------+----------------+-------------------------------------------------------------------+
+| paginate      | query          | *(optional)* Number of items to display per page, by default = 10 |
++---------------+----------------+-------------------------------------------------------------------+
+
+To see the first page of all channels use the below method:
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/channels/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json"
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 200 OK
+
+.. code-block:: json
+
+    {
+        "page": 1,
+        "limit": 10,
+        "pages": 1,
+        "total": 2,
+        "_links": {
+            "self": {
+                "href": "\/api\/v1\/channels\/?page=1&limit=10"
+            },
+            "first": {
+                "href": "\/api\/v1\/channels\/?page=1&limit=10"
+            },
+            "last": {
+                "href": "\/api\/v1\/channels\/?page=1&limit=10"
+            }
+        },
+        "_embedded": {
+            "items": [
+                {
+                    "id": 1,
+                    "code": "US_WEB",
```

Comment:
For channel index we should change a default serialization group to present slightly more information